### PR TITLE
[Instrumentation.Http] Move package from main repository

### PR DESCRIPTION
**Pull request by Kielek**
Towards https://github.com/open-telemetry/opentelemetry-dotnet/issues/5526

Leftover: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1693

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by matt-hensley — APPROVED** [review]

File: opentelemetry-dotnet-contrib.sln [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (vishweshbankwar) [inline opentelemetry-dotnet-contrib.sln:2]: nit: remove extra line

File: src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,548 @@
+# Changelog
+
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md:2]: ## Unreleased

File: src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,548 @@
...
+* Update `OpenTelemetry.Api.ProviderBuilderExtensions` to `1.8.1`.
+  * Update `OpenTelemetry.Api` to `1.8.1`.
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md:4]: do we need to mention `OpenTelemetry.Api` separately. We only target `OpenTelemetry.Api.ProviderBuilderExtensions`

**Review by vishweshbankwar — APPROVED** [review]
LGTM - left couple minor comments.

**Review by rajkumar-rangaraj — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md (hunk 3 above)
Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md:6]: It is transitive dependency. Entry is copied from SqlClient, before I copied it from another package referencing exactly this package.

We can fix it all at once in separate PR if you want.

**Pull request merged**